implants/eldritch/src/pivot.rs

```diff
@@ -65,9 +66,9 @@ fn methods(builder: &mut MethodsBuilder) {
         smb_exec_impl::smb_exec(target, port, username, password, hash, command)
     }     
     // May want these too: PSRemoting, WMI, WinRM
-    fn port_scan(this:  PivotLibrary, target_cidrs: Vec<String>, ports: Vec<i32>, portocol: String) ->  anyhow::Result<Vec<String>> {
+    fn port_scan<'v>(this:  PivotLibrary, starlark_heap: &'v Heap, target_cidrs: Vec<String>, ports: Vec<i32>, portocol: String, timeout:  i32) ->  anyhow::Result<Vec<Dict<'v>>> {
```

Comment:
double check 'v is necessary

Reply:
Nope it's a part of starlark auto heap magic.